optimize/backend/src/it/java/io/camunda/optimize/service/snapshot/BackupServiceIT.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.service.snapshot;
+
+import static io.camunda.optimize.dto.optimize.BackupState.INCOMPLETE;
+import static io.camunda.optimize.service.util.SnapshotUtil.getSnapshotNameForImportIndices;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.optimize.AbstractCCSMIT;
+import io.camunda.optimize.dto.optimize.rest.BackupInfoDto;
+import io.camunda.optimize.service.BackupService;
+import io.camunda.optimize.service.db.es.schema.ElasticSearchSchemaManager;
+import io.camunda.optimize.service.db.schema.IndexMappingCreator;
+import io.camunda.optimize.service.util.configuration.db.DatabaseBackup;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+public class BackupServiceIT extends AbstractCCSMIT {
```

Comment:
@KarpovIvan - I can't see this test actually running on the CI - am I missing something?

Reply:
<img width="794" alt="image" src="https://github.com/user-attachments/assets/c8799d55-a80a-498e-9815-114b67d46bc9">
https://github.com/camunda/camunda/actions/runs/10960818622/job/30436333429
Tests passed in previous commit.

Reply:
maybe I am just blind, or maybe it's just Friday 🙈